tests/test_workflow_base.py

```diff
@@ -86,34 +86,6 @@ def test_base_workflow_submit_items_success(
     assert json.dumps(expected_submission_summary) in caplog.text
 
 
-def test_base_workflow_submit_items_missing_row_raises_warning(
```

Comment:
I don't believe this scenario is still possible as we're instantiating from `ItemSubmissionDB.get_batch_items` but please correct me if I'm wrong!